shared/utils.js

```diff
@@ -3,7 +3,13 @@
 // TODO: make all of these functions configurable from the outside
 // the user should be able to manipulate those
 
-function getNamespace(service, stage) {
+function getNamespace(serverless) {
+  const namespace = serverless.service.provider.k8s?.namespace
```

Comment:
Unfortunately we can use this modern syntax yet,

It's introduced with `ES2020`, so still gaining adoption, and with `serverless` v1, we're still obliged to support Node.js v6 (so it's up to `ES2015` we can freely use)

Reply:
Sorry, I fixed it now.